### ISSUE TITLE
Email notification displays NaN as duration for cancelled builds

### ISSUE THREAD

**Issue by Martynas-P** [Shippable/support#1201]
the email notification sent after a build is cancelled shows NaN as the duration for each build.

![nan](https://cloud.githubusercontent.com/assets/2590908/6771264/9bd1322c-d0d2-11e4-98ba-323f4444c11f.png)

**Comment by manishas** [comment]
@Martynas-P we cannot repro this at our end. Does this happen consistently for canceled builds?

**Comment by Martynas-P** [comment]
@manishas I can't reproduce it anymore either. I took a look at the reports and build history where this problem happened and I saw that the status for the builds was inconsistent as well. In the email, the build status was Canceled

![image](https://cloud.githubusercontent.com/assets/2590908/6951750/7af985fa-d8b5-11e4-9eea-42fb1fb16c2e.png)

But on the website it says that the build failed

![image](https://cloud.githubusercontent.com/assets/2590908/6951787/b7a87c4a-d8b5-11e4-9110-f1caedc48fa3.png)

Maybe this will help you track the problem down.
